addon/utils/plugins/table/backspace-plugin.ts

```diff
@@ -0,0 +1,50 @@
+import { BackspacePlugin } from '@lblod/ember-rdfa-editor/editor/input-handlers/backspace-handler';
+import {Manipulation, ManipulationGuidance } from '@lblod/ember-rdfa-editor/editor/input-handlers/manipulation';
+import RawEditor from 'dummy/utils/ce/raw-editor';
+
+/**
+ *
+ * @class TableBackspacePlugin
+ * @module plugins/table
+ */
+export default class TableBackspacePlugin implements BackspacePlugin {
+  label = 'backspace plugin for handling table nodes'
+
+  guidanceForManipulation(manipulation: Manipulation, editor: RawEditor) : ManipulationGuidance | null {
+    const voidExecutor = {
+      allow: true,
```

Comment:
why not just disallow backspace and not provide an executor?

Reply:
I don’t fully know how this plugin system works, if I pass false there then nothing will be executed?

Reply:
See https://github.com/lblod/ember-rdfa-editor/blob/master/addon/editor/input-handlers/backspace-handler.ts#L394 , yes it should stop
(in general I think having a good grasp of that backspace method will help understand the plugin system a bit more)